test/test-RelationParser.cc

```diff
@@ -76,7 +76,7 @@ TEST_CASE("Relation identifier starting with non lower alphabet", "[relation]")
 
 TEST_CASE("Relation identifier containing capital letters", "[relation]")
 {
-    mdp::ByteBuffer source = "+ Relation: deLete";
```

Comment:
This shouldn't be changed, this test is explicitly checking relation identifiers with uppercase characters:

```
TEST_CASE("Relation identifier containing capital letters", "[relation]")
```